pkg/chartRepo/repository/ChartRepoRepository.go

```diff
@@ -123,6 +125,22 @@ func (impl ChartRepoRepositoryImpl) FindAllWithDeploymentCount() ([]*ChartRepoWi
 	return repo, err
 }
 
+func (impl ChartRepoRepositoryImpl) FindDeploymentCountByChartRepoId(chartId int) (int, error) {
+	var activeDeploymentCount int
+	query := "select count(jq.ia_id ) as deployment_count" +
+		" from chart_repo left join" +
+		" (select aps.chart_repo_id as cr_id ,ia.id as ia_id from installed_app_versions iav" +
```

Comment:
we can only make this work by only executing the subquery and putting the id where clause there only